docs/sources/administration/service-accounts/index.md

```diff
@@ -120,6 +120,10 @@ You can assign roles to a service account using the Grafana UI or via the API. F
 
 In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}), you can also [assign RBAC roles]({{< relref "../roles-and-permissions/access-control/assign-rbac-roles" >}}) to grant very specific permissions to applications that interact with Grafana.
 
+{{% admonition type="note" %}}
+Since Grafana 10.2.0, the `No Basic Role` is available for organization users or service accounts. This role has no permissions. Permissions can be granted with RBAC.
```

Comment:
nit: ```suggestion
Since Grafana 10.2.0, the `No Basic Role` is available for organization users or service accounts. This role has no permissions. Permissions can be granted with RBAC.
```

> Permissions can be granted with RBAC.

WDYT? RBAC is always available and i'd opt to remove this sentence

Reply:
Not only is the permission only available with RBAC, RBAC is an exclusive for enterprise. I felt the need to include it. Are you keen on removing it?